README.md

```diff
@@ -63,21 +65,21 @@ An _initial extension_ could be used to address some of the [undefined behaviors
 
 ## Specifying Community Extensions
 
+### Community vs Local Extensions
+
+Extensions submitted to this repository are considered community extensions and undergo a formal review and approval process. These differ from local extensions, which may be institution-specific and do not require OCFL Editor approval. Community extensions are meant to be broadly applicable and supported across implementations.
+
 ### Layout
 
-Community extensions MUST be written as GitHub flavored markdown files in the `docs` directory of this repository. The
-filename of an extension is based on its *Registered Name* with a `.md` extension.
+Community extensions MUST be written as GitHub-flavored markdown files in this repository's `docs` directory. The filename of an extension is based on its *Registered Name* with a `.md` extension.
 
-Extensions are numbered sequentially, and the *Registered Name* of an extension is prefixed with this 4-digit, zero-padded
-decimal number. The *Registered Name* should be descriptive, use hyphens to separate words, and have a maximum of 250
-characters in total.
+Extensions are numbered sequentially, and an extension's *Registered Name* is prefixed with this 4-digit, zero-padded decimal number. The *Registered Name* should be descriptive, use hyphens to separate words, and have a maximum of 250 characters.
 
-New extensions should use `NNNN` as a place-holder for the next available prefix number at the time of merging. New extension pull-requests should not update the index document (`docs/index.md`), this will be done post-approval.
+New extensions should use `NNNN` as a placeholder for the next available prefix number at the time of merging. New extension pull-requests should not update the index document (`docs/index.md`); this will be done after approval.
 
-Extensions are intended to be mostly static once published. Substantial revisions of content beyond simple fixes warrants publishing a new extension, and marking the old extension obsolete by updating the *Obsoletes/Obsoleted by* sections in each extension respectively.
+Once published, extensions are intended to be mostly static. Substantial revisions of content beyond simple fixes warrant publishing a new extension and marking the old extension obsolete by updating the *Obsoletes/Obsoleted by* sections in each extension, respectively.
```

Comment:
I think while we are making changes we should say "Once published, extensions are intended to be static." (ie. remove "mostly")